### ISSUE TITLE
Новые голоса.

### ISSUE THREAD

**Issue by OXEPU3M** [GaijinEntertainment/fmod_studio_warthunder_for_modders#211]
Во-первых, рад, что вы проделали столь масштабную работу, и хочу поблагодарить, что мы увидели её так скоро и в этом патче.

Однако, возник ряд вопросов:
1. Что это? И почему при этом прячется в банке с эффектами?
![noise](https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/74115508/2275fd82-be50-49d7-a73e-d973398f3773)

2. Что такое параметр generation? И как он применяется?
3. Я так и не понял как слушать файлы на таймлайне. Вы говорили, нужно запустить снапшот, но ни команда, ни помещение его прямо в ивент, не даёт ничего кроме тишины.
Хоть картинку нарисуйте, буду премного благодарен.
4. Почему, если положить в ивент чистый голос .wav качество сильно режется, даже если вырезать все ивенты с background noises и параметр generation?

**Comment by OXEPU3M** [comment]
Так, с четвёртым разобрался - глобальные бусы и снапшоты. Интересное решение.
Но лучше уж, дайте картинку, на всякий, вдруг упустил чего.
Спаме.

**Comment by Uchuujin01** [comment]
Привет!
1 - это фоновые статические шумы для воспроизведения подкладом под голоса экипажа.
2 - generation это вспомогательный параметр для контроля за инстансами ивентов из кода, его лучше оставить для корректного воспроизведения ивентов.
3 - Имеется ввиду как слушать ивенты диалогов? Или интерьерные звуки, например стрельбы? Если второе, то надо запустить снапшот tank, выбрать его во вкладке Snapshots в Mixer VIew, и нажать снепшоту Play. Все, можно запускать интерьерные звуки. Если речь о диалогах, то ивенты диавлогов нельзя послушать из Fmod, поскольку в ивентах не вавки а ссылки на вавки с помощью Programmer Instruments.
4 - да, там фильтр на основе конволююционного плеера импульсов, который есть в Fmod в виде плагина.

**Comment by Uchuujin01** [comment]
> Во-первых, рад, что вы проделали столь масштабную работу, и хочу поблагодарить, что мы увидели её так скоро и в этом патче.

Спасибо! Уверен, нас ждут интересные голосовые моды на основе новой системы )

**Comment by OXEPU3M** [comment]
@Uchuujin01 


> Если речь о диалогах, то ивенты диавлогов нельзя послушать из Fmod, поскольку в ивентах не вавки а ссылки на вавки с помощью Programmer Instruments.

Про ссылки и так понятно. Но только если положить вместо них wav файл, тоже ничего не происходит.
И касательно выстрелов. Проигрываются они только со снапшотом tank_open, просто tank не даёт результата.
Диалоги же не работают ни с тем, ни с другим.

Я чёт, запутался.

**Comment by OXEPU3M** [comment]
Встретил новую напасть.
Удалил все background_noise и вообще воткнул их в банк с голосами.
Удалили все эффекты из снапшотов. Так же зачистил ивенты.
На самый последний случай вообще взял ВСЕ голосовые ивенты из всех подгрупп и перевёл в корневую группу dialog в миксере.

Но почему-то всё равно слышу посторонние шумы в игре при работе озвучке. Где я что упустил?

**Comment by OXEPU3M** [comment]
> Но почему-то всё равно слышу посторонние шумы в игре при работе озвучке. Где я что упустил?

Короче. Без банка tank_effects в папке mod, шумы появляются в любом случае.
Очень страшно. Мы не знаем, что это такое....
